include/envoy/http/filter.h

```diff
@@ -71,9 +71,9 @@ class StreamFilterCallbacks {
   virtual ~StreamFilterCallbacks() {}
 
   /**
-   * @return uint64_t the ID of the originating connection for logging purposes.
+   * @return const Network::Connection* the originating connection.
    */
-  virtual uint64_t connectionId() PURE;
```

Comment:
Please leave around the connectionId() function and mark it for deprecation. We will remove it in 1.5.0 release cycle.